packages/services/f-analytics/src/store/analyticsModule.js

```diff
@@ -0,0 +1,32 @@
+export default {
+    namespaced: true,
+
+    state: () => ({
+        platformData: {
+            environment: '',
+            name: '',
+            appType: '',
+            applicationId: 0,
+            userAgent: '',
+            branding: '',
+            country: '',
+            language: '',
+            jeUserPercentage: 0,
+            currency: '',
+            version: '',
+            instancePosition: ''
+        }
+    }),
+
+    actions: {
+        updatePlatformData: ({ commit }, platformData) => {
+            commit('updatePlatformData', platformData);
```

Comment:
Do we want a `mutations` constants file for this `updatePlatformData`

Reply:
I thought it was a bit of overkill atm but if it extends into a more richer $store in the future then would be good to add then.